tests/pytest/test_dynamicschema.py

```diff
@@ -1,21 +1,22 @@
 """This testing class tests whether we can run a workload by defining a schema dynamically."""
 
 import palimpzest as pz
+import palimpzest.schema_builder as sb
 from palimpzest.constants import Model
 
 data_path = "tests/pytest/data/"
 
 
 def test_dynamicschema_jsonld():
-    clinical_schema = pz.SchemaBuilder.from_file(data_path + "synapse_schema.jsonld", schema_type=pz.TextFile)
+    clinical_schema = pz.sb.SchemaBuilder.from_file(data_path + "synapse_schema.jsonld", schema_type=pz.TextFile)
```

Comment:
With the new import on line 4, would this now just become `sb.SchemaBuilder...`?